client/handlers/getstudent.js

```diff
@@ -0,0 +1,61 @@
+
+
+ export class students {
+    student = '';
+    LastName = '';
+    module = [];
+    class=''
+
+    constructor(data){
+        Object.assign(this,data)
+       
+        
+    }
+    render(){
+         const container = document.createElement('div');
+        container.innerHTML = ` <h1> Here is students' Information </h1> \n student: ${this.student} ${this.LastName}`;
+        container.className = 'container';
+
+         const classname = document.createElement('p')
+         classname.innerHTML = `class: ${this.class}`
+        
+
+        const renderedmoduls = this.module
+            renderedmoduls.forEach(mode=>{
+                const m = document.createElement('div');
+                m.innerHTML =`model: ${mode}`;
+                m.id = 'mode';
+             container.appendChild(m);
+            })
```

Comment:
I would do here like this instead of create  divs  with same id => ```
const renderedmoduls = this.module;
		const m = document.createElement('div');
		m.innerHTML = `model:`;
		m.id = 'mode';
		renderedmoduls.forEach((mode) => {
			m.innerHTML += ` ${mode}`;
		});
		container.appendChild(m);
```

Reply:
you mean putting both mode's in one div right??

Reply:
resolved!